net/websocat/Portfile

```diff
@@ -15,6 +16,8 @@ description         Netcat, curl and socat for WebSockets.
 long_description    Command-line client for web sockets, like \
                     netcat/curl/socat for ws://. 
 
+build.args          --features=ssl
```

Comment:
So does this now require a dependency added for `openssl`?

Reply:
Looking at the Rust libraries (`native-tls`, `openssl`, `openssl-sys` and others), looks like it does make sense to include the dependency; added.

Reply:
Normally `openssl` is a library dependency. Is is being a build dependency some aspect of cargo that I don't understand? Also, can libressl work here? If so, we normally write this dependency as follows to allow it:

`path:lib/libssl.dylib:openssl
`

Reply:
I'm not exactly sure, but the resulting binary does not show any linkage to any SSL lib when examined with `otool`. 

```
$ otool -L `which websocat`
/opt/macports-test/bin/websocat:
        /System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 58286.255.3)
        /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1575.17.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.250.1)
        /usr/lib/libresolv.9.dylib (compatibility version 1.0.0, current version 1.0.0)
```

I think that there is some bit of static linking going on during the cargo build, hence why I initially went with `depends_build`.  However since I'm not sure, I've opted for safe changes and added the `openssl` dependency as a library dependency.

Reply:
Sounds good. If websocat needs it at build time and, presumably, at run time, then a library dependency classification is appropriate.